eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/init/StartupJob.java

```diff
@@ -15,254 +16,255 @@
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.Status;
+import org.eclipse.core.runtime.jobs.IJobChangeEvent;
 import org.eclipse.core.runtime.jobs.Job;
 import org.eclipse.ui.IPageLayout;
 import org.eclipse.ui.ISelectionService;
 import org.eclipse.ui.IWorkbench;
 import org.eclipse.ui.IWorkbenchWindow;
 import org.eclipse.ui.PlatformUI;
+
 import cc.codechecker.plugin.CodeCheckerNature;
+import cc.codechecker.plugin.ExternalLogger;
+import cc.codechecker.plugin.Logger;
 import cc.codechecker.plugin.config.CcConfiguration;
 import cc.codechecker.plugin.config.CodeCheckerContext;
-
-import cc.codechecker.plugin.Logger;
-import cc.codechecker.plugin.ExternalLogger;
-
+import cc.codechecker.plugin.report.job.AnalyzeJob;
+import cc.codechecker.plugin.report.job.JobDoneChangeListener;
+import cc.codechecker.plugin.report.job.PlistParseJob;
 import cc.codechecker.plugin.runtime.SLogger;
-import cc.codechecker.plugin.runtime.CodecheckerServerThread;
 
+/**
+ * This eclipse job is responsible for registering the Build listener.
+ * @author vodorok
```

Comment:
Usually we do not add the username to the comments.

Reply:
Did a quick grep, removed all occurances.